include/fastdds/rtps/transport/TransportDescriptorInterface.h

```diff
@@ -33,17 +33,24 @@ class TransportInterface;
  * */
 struct TransportDescriptorInterface
 {
-    TransportDescriptorInterface(uint32_t maximumMessageSize, uint32_t maximumInitialPeersRange)
+    TransportDescriptorInterface(
+            uint32_t maximumMessageSize,
+            uint32_t maximumInitialPeersRange)
         : maxMessageSize(maximumMessageSize)
         , maxInitialPeersRange(maximumInitialPeersRange)
-    {}
+    {
+    }
 
-    TransportDescriptorInterface(const TransportDescriptorInterface& t)
+    TransportDescriptorInterface(
+            const TransportDescriptorInterface& t)
         : maxMessageSize(t.maxMessageSize)
         , maxInitialPeersRange(t.maxInitialPeersRange)
-    {}
+    {
+    }
 
-    virtual ~TransportDescriptorInterface(){}
+    virtual ~TransportDescriptorInterface()
+    {
+    }
```

Comment:
Consider using the rule of three here. 

```
    TransportDescriptorInterface(
            const TransportDescriptorInterface& t) = default;

    TransportDescriptorInterface& operator=(
            const TransportDescriptorInterface& t) = default;

    virtual ~TransportDescriptorInterface() = default;
```

Reply:
Done in 4fcfb6feaa85204124ab730ef1a58d1ee5337c57